app/scenes/Settings/Slack.js

```diff
@@ -136,7 +136,7 @@ const ListItem = styled.li`
 const Code = styled.code`
   padding: 4px 6px;
   margin: 0 2px;
-  background: #eaebea;
+  background: #2c2b2c;
```

Comment:
I think this might break light mode? I believe this should take into consideration theme

Reply:
`props.theme.codeBackground` should do it